sdk/servicebus/service-bus/src/util/constants.ts

```diff
@@ -9,3 +9,264 @@ export const packageJsonInfo = {
 export const messageDispositionTimeout = 20000;
 
 export const max32BitNumber = Math.pow(2, 31) - 1;
+
+/**
+ * Defines constants for use with service bus.
+ *
+ * @const
+ * @type {string}
+ */
+export const ServiceBusAtomXmlConstants = {
```

Comment:
There are a few constants here that aren't being used anywhere
- MAX_SUBSCRIPTIONS_PER_TOPIC
- MAX_SQL_FILTERS_PER_TOPIC
- MAX_CORRELATION_FILTERS_PER_TOPIC
- DEFAULT_RULE_DESCRIPTION
- DEFAULT_RULE_NAME
- WRAP_ACCESS_TOKEN
- WRAP_ACCESS_TOKEN_EXPIRES_UTC
- WRAP_ACCESS_TOKEN_EXPIRES_IN
- API_VERSION_QUERY_KEY
- CURRENT_API_VERSION

Reply:
Some of the above constants are being used in the latest update, but there are still some that aren't being used

Reply:
After latest update, there are still constants that are not being used

Reply:
Removed unused ones - in general, the specific use of fields, parameters can be updated to tighten the types. But that would involve a lot more testing, investigation and can be updated later as well